### DISCUSSION TITLE
Labs 1.10.70 isn't working with H13 lens mods

### DISCUSSION THREAD

**Discussion by SimonRSMW** [gopro/labs#1071] [General]
I recently installed Labs 1.10.70 onto a HERO13. I've been finding that if I attach a lens mod, such as the Macro, the camera will start up, then give me a pink screen, then reboot itself, and then continues to do this in a loop until I remove the lens mod. Anyone else finding this? I've reset the camera and reinstalled the factory firmware, then tried a new download and install of Labs, but it does the same thing.

**Comment by dnewman-gpsw** [comment]
As Labs does support the lens mods, do a factory reset (not a firmware reinstall) they should clear out any configuration errors.

**Reply by ardenjacoby** [reply]
read this forum https://github.com/gopro/labs/discussions/1061 I had the same issue. unfortunately labs has a bug for the 13

**Reply by SimonRSMW** [reply]
> read this forum #1061 I had the same issue. unfortunately labs has a bug for the 13

Thanks. That's a similar issue, although mine only seems to do it with the lens mods attached. If I remove them, it's stable.
